api/src/operations/request/index.ts

```diff
@@ -19,27 +19,28 @@ export default defineOperationApi<Options>({
 				return acc;
 			}, {} as Record<string, string>) ?? {};
 
-		if (!customHeaders['Content-Type'] && isValidJSON(body)) {
+		if (!customHeaders['Content-Type'] && isValidJSON(String(body))) {
```

Comment:
I did not change this behavior it just got moved into shared utils. I believe the body always comes in as a string because before `isValidJSON(value: any)` tests by using `parseJSON(input: string)` which just uses JSON.parse().

Will the operation ever get passed an object or will it always get passed stringified JSON?